app/migrations/migrations/3_init_newrl_tokens.py

```diff
@@ -0,0 +1,52 @@
+import sqlite3
+
+from ...constants import NEWRL_DB
+
+FOUNDATION_WALLET = '0xc29193dbab0fe018d878e258c93064f01210ec1a'
+ASQI_WALLET = '0x20513a419d5b11cd510ae518dc04ac1690afbed6'
+FOUNDATION_PUBLIC_KEY = 'sB8/+o32Q7tRTjB2XcG65QS94XOj9nP+mI7S6RIHuXzKLRlbpnu95Zw0MxJ2VGacF4TY5rdrIB8VNweKzEqGzg=='
+ASQI_PUBLIC_KEY = 'PizgnsfVWBzJxJ6RteOQ1ZyeOdc9n5KT+GrQpKz7IXLQIiVmSlvZ5EHw83GZL7wqZYQiGrHH+lKU7xE5KxmeKg=='
+
+FOUNDATION_RESERVE = 1500000000
+
+def migrate():
+    init_newrl_tokens()
+
+def init_newrl_tokens():
+    """Initialise NWRL tokens and pay the foundation"""
+    con = sqlite3.connect(NEWRL_DB)
+    cur = con.cursor()
+
+    credit_wallet(cur, ASQI_WALLET, FOUNDATION_RESERVE)
+    credit_wallet(cur, FOUNDATION_WALLET, FOUNDATION_RESERVE)
+
+    con.commit()
+    con.close()
+
+
+def credit_wallet(cur, wallet, amount):
```

Comment:
This won't work twice.. we need to use the add_token function from db_updater
The second time onwards use of credit_wallet should not re-insert line 40.
All of this is taken care of in the add_token function.